### DISCUSSION TITLE
about tokenizer

### DISCUSSION THREAD

**Discussion by emjay73** [seominjoon/kaist-ai605#109] [Q&A] [answered]
Is it ok to use "nltk.tokenize.word_tokenize"?
So that I can separate punctuation?

**Comment by sorobedio** [comment]
Just after the Search class you can read:
You will use the same space-based tokenizer that you used in Assignment 1, with lowercasing to make it case insensitive.

**Reply by emjay73** [reply]
Thank you for the reminder.
I saw that too but just wondered whether I have a chance to use the new library.
But thanks!

**Comment by seominjoon** [comment, marked as answer]
It's okay to use other tokenizers, but it will be easier for me and other people to debug it if you use the same space tokenizer.
